frontends/p4/removeReturns.h

```diff
@@ -41,6 +41,58 @@ class HasExits : public Inspector {
     void postorder(const IR::ReturnStatement *) override { hasReturns = true; }
 };
 
+/**
+This visitor rewrites code after an 'if' that does a branch by moving the
```

Comment:
By `branch` you presumably mean `exit`, `return`, `continue`, right? Please add that to the comment as when `branch` is said, I imagine `if` or `switch`.

Reply:
Yes, this is confusing.  I rewrote the comment to use the term "jump statement" (as that is what the C spec uses for these) instead of "branch"

Reply:
Thansk, that is a better name :-).